src/common/models/project.go

```diff
@@ -120,11 +124,71 @@ func (p *Project) AutoScan() bool {
 	return isTrue(auto)
 }
 
+// QueryPublic ...
+func (p *Project) QueryPublic(qs orm.QuerySeter, key string, value interface{}) orm.QuerySeter {
```

Comment:
what is the `key` for？

Reply:
`key` is the raw key in the `q.Query.KeyWords`